source/install/troubleshooting.rst

```diff
@@ -1,16 +1,11 @@
 ..  _troubleshooting:
 
-Troubleshooting
```

Comment:
Why do we remove the title?

Reply:
I moved the document within the administrator.rst file (from Installation to its own section called "Troubleshooting" . Doing that duplicated "Troubleshooting" as a header and then the title of the page was a sub-header, with the document contents being a third level of headers (among other formatting issues). Changing this and some of the heading formatting fixed the issue. 

Hope that makes sense and if there is a less convoluted way to have done that let me know! :)